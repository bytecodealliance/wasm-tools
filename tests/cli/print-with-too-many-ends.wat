;; RUN: print %

(module
  (func end i32.const 0 drop end)
)
