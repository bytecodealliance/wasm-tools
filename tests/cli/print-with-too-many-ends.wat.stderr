error: operators remaining after end of function body or expression (at offset 0x18)
